src/oemof/solph/blocks/bus.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+
+"""Creating sets, variables, constraints and parts of the objective function
+for Bus objects.
+
+SPDX-FileCopyrightText: Uwe Krien <krien@uni-bremen.de>
+SPDX-FileCopyrightText: Simon Hilpert
+SPDX-FileCopyrightText: Cord Kaldemeyer
+SPDX-FileCopyrightText: Patrik Schönfeldt
+SPDX-FileCopyrightText: Birgit Schachler
+SPDX-FileCopyrightText: jnnr
+SPDX-FileCopyrightText: jmloenneberga
```

Comment:
These copyright notes are taken from blocks.py. Some of these authors might not have contributed to the new blocks/bus.py. Should we adjust this? (Might be tedious in some cases.)